qiskit_experiments/library/characterization/fine_2q_amplitude.py

```diff
@@ -0,0 +1,110 @@
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Fine amplitude characterization experiment for two-qubit gates."""
+
+from typing import Optional, Tuple
+import numpy as np
+
+from qiskit import QiskitError
+from qiskit.circuit import QuantumCircuit, Gate
+from qiskit.providers.backend import Backend
+
+from qiskit_experiments.library.characterization.fine_amplitude import FineAmplitude
+from qiskit_experiments.framework import Options
+
+
+class FineTwoQubitAmplitude(FineAmplitude):
+    r"""A fine amplitude experiment with all the options set for a two-qubit `R(theta)` gate.
+
+    # section: overview
+
+        :class:`FineTwoQubitAmplitude` is a subclass of :class:`FineAmplitude` and is used
+        to set the appropriate values for the default options. The following example
+        illustrates how this experiment can be used
+
+        .. code:: python
+
+            inst_map = InstructionScheduleMap()
+            inst_map.add("rzx", qubits, my_rzx_schedule)
+
+            fine_amp = FineTwoQubitAmplitude(qubits, RZXGate(np.pi/2), backend)
+            fine_amp.set_transpile_options(inst_map=inst_map)
+
+            exp_data = fine_amp.run()
+
+        Here, the user provides the schedule for the `RZXGate` by passing it to the
+        experiment using the instruction schedule map in the transpile options. The example
+        above will create the circuits
+
+        .. parsed-literal::
+
+                 ┌────────────┐     ┌────────────┐
+            q_0: ┤0           ├ ... ┤0           ├───
+                 │  rzx(1.57) │     │  rzx(1.57) │┌─┐
+            q_1: ┤1           ├ ... ┤1           ├┤M├
+                 └────────────┘     └────────────┘└╥┘
+            c: 1/══════════════ ...  ══════════════╩═
+                                                   0
+
+    """
+
+    def __init__(self, qubits: Tuple[int, int], gate: Gate, backend: Optional[Backend] = None):
+        """Initialize the experiment.
+
+        Args:
+            qubits: A length two tuple of the qubits on which to run.
+            gate: A Qiskit Gate instruction with one parameter corresponding to the rotation angle.
+            backend: The backend to run on.
+
+        Raises:
+            QiskitError: If the gate does not have exactly one parameter.
+        """
+
+        if len(gate.params) != 1:
+            raise QiskitError(f"A gate with a single rotation angle is required.")
+
+        angle = gate.params[0]
+        label = gate.name + f"({angle})"
+        super().__init__(qubits, Gate(gate.name, 2, [], label=label), backend=backend)
```

Comment:
The issue is that `RZXGate` has a parameter whereas we a dealing with a gate that does not have a parameter. This can cause pulse issues with circuit.calibrations